spec/spec_helper.rb

```diff
@@ -1,7 +1,16 @@
 require "simplecov"
-require 'coveralls'
 
-SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+if ENV["CI"]
```

Comment:
What if instead of using `CI` environment variable there could be an environment variable something like `LCOV_REPORT_PATH` which could be also used at https://github.com/watir/watir-rails/pull/31/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR41 - basically `ci.yml` could define this variable and it could be used here as well (first check if it is defined and then use it as path on line 8). This would allow running specs (I hope?) also on your own machine with specifying this coverage file path and it would be in sync with ci.yml and this file (otherwise if either place is modified then things break).

Reply:
Done :+1:.

Reply:
Okay, it somehow doesn't work https://github.com/ojab/watir-rails/actions/runs/525421638, investigating and will comment when it will be fixed.

Reply:
Fixed, https://github.com/ojab/watir-rails/actions/runs/525445060.